extensions/smallrye-graphql-client/runtime/src/main/java/io/quarkus/smallrye/graphql/client/runtime/GraphQLClientConfigurationMergerBean.java

```diff
@@ -80,6 +113,13 @@ private GraphQLClientConfiguration toSmallRyeNativeConfiguration(GraphQLClientCo
         return transformed;
     }
 
+    private String getTestingServerUrl() {
+        Config config = ConfigProvider.getConfig();
+        // the client extension doesn't have dependencies on neither the server extension nor quarkus-vertx-http, so guessing
+        // is somewhat limited
+        return "http://localhost:" + config.getOptionalValue("quarkus.http.test-port", int.class).orElse(8081) + "/graphql";
```

Comment:
I welcome suggestions how to improve it. Because we don't want to introduce dependencies on `quarkus-vertx-http` or `quarkus-smallrye-graphql` extensions just for this, we can't directly read their configuration, and they might not be present at all

Reply:
Can you not get this from a BuildItem (and pass in here via some recorder ?)

Reply:
The problem is the non-existent dependencies, such BuildItem would probably have to exist in some shared module (Quarkus core probably), and I'm not so sure we want to complicate stuff this much just for this, and add graphql-specific stuff into Quarkus core. In 98 % of cases this should work, and we also log a message explaining how to fix it if it doesn't work.